packages/api/cms-api/src/blob-storage/backends/blob-storage-backend.interface.ts

```diff
@@ -16,8 +18,8 @@ export interface BlobStorageBackendInterface {
     removeFolder(folderName: string): Promise<void>;
     fileExists(folderName: string, fileName: string): Promise<boolean>;
     createFile(folderName: string, fileName: string, data: NodeJS.ReadableStream | Buffer | string, options: CreateFileOptions): Promise<void>;
-    getFile(folderName: string, fileName: string): Promise<NodeJS.ReadableStream>;
-    getPartialFile(folderName: string, fileName: string, offset: number, length: number): Promise<NodeJS.ReadableStream>;
+    getFile(folderName: string, fileName: string): Promise<Readable>;
+    getPartialFile(folderName: string, fileName: string, offset: number, length: number): Promise<Readable>;
```

Comment:
This change from `ReadableStream` to `Readable` is not a breaking change because `Readable` implements `ReadableStream`:

<img width="633" alt="Bildschirmfoto 2024-11-07 um 00 05 18" src="https://github.com/user-attachments/assets/8672ec70-65c4-49ca-bf36-834bed939fd6">